### ISSUE TITLE
Depedency on geerlingguy.repo-epel

### ISSUE THREAD

**Issue by steveraden** [nmusatti/docker-ce#1]
Nicola, Thanks so much for your work. `source_python` helped my life this morning (on Cent OS 8 Stream) and then I tried `docker-ce.`   Minor issue, I think a requirements file is needed because I had to manually install geerlingguy..repo-epel.  Without that step, I get this error. But I'm new to Ansible so maybe I am wrong.  

```
The error appears to be in '/Users/stever/.ansible/roles/nmusatti.docker_ce/tasks/redhat/install.yml': line 7, column 11, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  include_role:
    name: geerlingguy.repo-epel
          ^ here
```

**Comment by nmusatti** [comment]
Steven, I'm sorry, but somehow I must have missed the notification for your issue. There was a reason why I didn't include an explicit requirement, but at the moment I can't remember what it was. I'll try and do some tests as soon as I find the time.

**Comment by nmusatti** [comment]
The problem with the geerlingguy.repo-epel role is that is only required for RHEL and derived distros, so I don't think it would be a good idea to impose it on users of Ubuntu or Fedora.

**Comment by grafodomes** [comment]
Fair enough.  I'm not supporting whatever that was related to anymore.  Thanks for responding though.

**Comment by nmusatti** [comment]
Sorry for taking so long.
